app/models/author.rb

```diff
@@ -1,3 +1,8 @@
 class Author < ActiveRecord::Base
   validates :name, presence: true
+  has_many :books
+
+  def popular_books
```

Comment:
I think is a good idea to extract this number as a default (like `popular_books(limit=5)`) and allow the views layer to ask for more when necessary but without delegate it as a view responsibility, what you think?

Also, I see it as a WIP method and in order to really get the "popular books" I believe we should calculate the ratings and the number of reviews to define the top books.